playwright/lib/testUtils.ts

```diff
@@ -1,5 +1,11 @@
+import { test } from "@playwright/test";
 import { createServer, IncomingMessage, ServerResponse } from "http";
 
+export function todo(title: string) {
+  // eslint-disable-next-line @typescript-eslint/no-empty-function
+  test.skip(title, () => {});
```

Comment:
```suggestion
  test.fixme(title, () => {});
```
https://playwright.dev/docs/api/class-test#test-fixme

Reply:
Fixme is introducing side-effects:

![image](https://user-images.githubusercontent.com/3504472/146223810-b45525b5-f464-4ce2-aac1-2cc0fa3804a4.png)

When it should be:

![image](https://user-images.githubusercontent.com/3504472/146223868-ab5d100c-2e24-4d43-b538-9344acdaa42c.png)